### PR TITLE
[ADD] Adding fields sequence for model acceptability criteria

### PR DESCRIPTION
**Pull request by egrivero**
FIX vauxoo/instance#180
Close vauxoo/instance#265

### CODE REVIEW

File: user_story/model/user_story.py [hunk 1]
```diff
@@ -633,15 +633,35 @@ def _get_us_ca_numbers(self, cr, uid, ids, fieldname, arg,
...
+        elif len(ac_ids):
+            order = [x[1] for x in ac_ids]
+            read_ac = self.read(cr, uid, order,
+                                ['sequence_ac'], context=context)
+            maxi = max(read_ac, key=lambda x: x['sequence_ac'])
+            seq = maxi['sequence_ac'] + 1
+        else:
+            seq = 1
+        return seq
+
```
Comment (nhomar) [inline user_story/model/user_story.py:662]: this method should be:
variable seq and acc_number are defined inside IF's this is a bad practice, you need initialize always the default value before go to if.

Plase check that.

thanks.

Comment (egrivero) [inline user_story/model/user_story.py:662]: the variables are defined in both the if and the else, even so must be initialized ?. I understand when used alone in a condition if

Comment (nhomar) [inline user_story/model/user_story.py:662]: Yes.

The correct approach is:

``` python
var=1
if something:
   var=logic()
if other:
   var=logic2()
```

Use:

``` python
if something:
    var=logic()
elif other:
    var=logic2()
else:
    var=1
```

Is not considered clean.

Why?

The default behavior should be shown first and then the 1st case possible and then the second it is more readable than left the default value at the end in the else (yes I know python shit.!)

**Comment by egrivero** [comment]
Travis generates errors but are not related to the changes I make

**Pull request merged**